src/lib/orionld/temporal/temporalTenantInitialise.cpp

```diff
@@ -208,6 +208,19 @@ bool temporalTenantInitialise(const char* tenant)
     {
       LM_K(("Connection is ok with the %s database\n", tenant));
       LM_K(("Now crreating the tables for the teanant %s \n", tenant));
+
+      // need to create a routine - Fix me PLEEEEEASE - start
+      PGresult *res = PQexec(oldPgDbTenantConnection,
+        "SELECT entity_id FROM entity_table");
+
+      if( PQresultStatus(res)==PGRES_FATAL_ERROR )
+        {
+          LM_K(("CCSR - Table does not EXIST - what can we do"));
+        }
+
+      PQclear(res);
+      // need to create a routine - Fix me PLEEEEEASE - end
```

Comment:
So, let's create a routine for it.
We can leave this "unorthodox" way of detecting whether a DB exists or not for now, with a FIXME for the future.

The call could look like this:
```
if (temporalDbExists(tenant) == true)
  return true;
```

Reply:
Fixed in ...